backend/get_image.py

```diff
@@ -0,0 +1,33 @@
+from flask import Blueprint, request, send_file
+import io
+from utils import Error
+import validation
+import json
+import model
+
+get_image_blueprint = Blueprint("get_image", __name__)
+
+
+@get_image_blueprint.route('/get_image', methods=['GET'])
+def get_image():
+    '''
+    Body: {"upc"}
```

Comment:
The get_image() function should be better documented with comments to explain exactly what it does.

Reply:
Good point. More explanation has been added.